City-of-Helsinki/helsinki-paatokset
# PP-154: Install Paatokset Ahjo module and create content / entity types

**To test**
* Run `make build; make post-install; drush-cr;`
* Make sure that the module can be found in the modules list and is enabled: https://helsinki-paatokset.docker.so/en/admin/modules
* Test that the integrations can be run according to the instructions found in: https://github.com/City-of-Helsinki/paatokset_ahjo
* Make sure that tests run successfully: `make test`
